src/components/Tracker/elements/Encounters/Encounters.tsx

```diff
@@ -119,6 +142,33 @@ const Encounters = React.memo(function Encounters() {
           <div className={styles.header}>
             <span className={styles.location}>{encounter.location}</span>
             <div className={styles.buttons}>
+            {isTrainerRoute && 
+            <div
+              className={styles.question}
+              data-testid={`badge-detail-${trainerIndex + 13}`}
+              onClick={(e) => handleOpen(e, trainerIndex + 13)}
```

Comment:
Why 13?

Reply:
There are 13 Gym Leaders/ E4 members. I'll make that a constant so it's a little more obvious